tests/conftest.py

```diff
@@ -2176,7 +2191,12 @@ def bucket_class_factory(request, mcg_obj, backingstore_factory):
             a bucketclass
```

Comment:
Add info when it returns None to docstring.

Reply:
Done